csrankings-e.csv

```diff
@@ -348,6 +348,7 @@ Elvin Isufi,TU Delft,https://sites.google.com/site/elvinisufihp,wvywFdwAAAAJ
 Ely Porat,Bar-Ilan University,http://u.cs.biu.ac.il/~porately,uqTfLEgAAAAJ
 Elzbieta Hajnicz,IPI PAN,https://ipipan.waw.pl/instytut/pracownicy/elzbieta-hajnicz,UH3lJT8xzIcC
 Elín Carstensdóttir,Univ. of California - Santa Cruz,https://games.arts.ucsc.edu/people/elin-carstensdottir,fO6FFqAAAAAJ
+Emad Boctor,Johns Hopkins University,https://musiic.lcsr.jhu.edu/,17P8v9QAAAAJ

```

Comment:
<img width="1230" alt="Screen Shot 2024-11-04 at 10 43 01 AM" src="https://github.com/user-attachments/assets/00b55431-7b7e-413e-8189-eb07b0589d2a">

https://www.cs.jhu.edu/people/cs-affiliates/joint-and-secondary-faculty/